Buildings/Resources/Documentation/userGuide/build/html/workArounds.html

```diff
@@ -322,7 +329,7 @@ <h2>3.5. Avoiding overspecified initialization problems<a class="headerlink" hre
     </p>
     <p>
         &copy; Copyright 2012-2020, The Regents of the University of California (through Lawrence Berkeley National Laboratory).<br/>
-      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 2.1.2.<br/>
+      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.8.5.<br/>
```

Comment:
It looks like you have an old version of sphinx.

Reply:
planning to update